warp: update WARP definitions (#5728)
cloudflare/cloudflare-docs: content/warp-client/warp-modes.md

## WARP+

> While WARP is able to take advantage of the many Cloudflare data centers around the world to give you a more private and robust connection, WARP+ subscribers get access to a larger network. More cities to connect to means you’re likely to be closer to a Cloudflare data center – which can reduce the latency between your device and Cloudflare and improve your browsing speed. As a result, sites load faster, both for those on the Cloudflare network and those that aren’t.

Reviewer: ```suggestion
While WARP is able to take advantage of the many Cloudflare data centers around the world to give you a more private and robust connection, WARP+ subscribers get access to a larger network. More cities to connect to means you are likely to be closer to a Cloudflare data center – which can reduce the latency between your device and Cloudflare, and improve your browsing speed. As a result, sites load faster, both for those on the Cloudflare network and those that are not.
```
